core/services/feeds/orm_test.go

```diff
@@ -206,11 +206,12 @@ func Test_ORM_CreateChainConfig(t *testing.T) {
 		orm  = setupORM(t)
 		fmID = createFeedsManager(t, orm)
 		cfg1 = feeds.ChainConfig{
-			FeedsManagerID: fmID,
-			ChainID:        "1",
-			ChainType:      feeds.ChainTypeEVM,
-			AccountAddress: "0x0001",
-			AdminAddress:   "0x1001",
+			FeedsManagerID:          fmID,
+			ChainID:                 "1",
+			ChainType:               feeds.ChainTypeEVM,
+			AccountAddress:          "0x0001",
+			AdminAddress:            "0x1001",
+			AccountAddressPublicKey: null.StringFrom("123"),
```

Comment:
the public key would also be in hex format ("0x...."). might be helpful to clarify that by having "0x0100" as an example.